tests/core/test_providers_manager.py

```diff
@@ -174,6 +172,13 @@
     'extra__yandexcloud__public_ssh_key',
     'extra__yandexcloud__service_account_json',
     'extra__yandexcloud__service_account_json_path',
+    'extra__snowflake__account',
+    'extra__snowflake__warehouse',
+    'extra__snowflake__database',
+    'extra__snowflake__region',
+    'extra__snowflake__aws_access_key_id',
+    'extra__snowflake__aws_secret_access_key',
+
```

Comment:
```suggestion
```

This is causing Static check failure: https://github.com/apache/airflow/pull/14724/checks?check_run_id=2089572707#step:9:270